server/monitor/src/main/java/org/apache/accumulo/monitor/rest/gc/GarbageCollectorStats.java

```diff
@@ -19,42 +19,40 @@
 package org.apache.accumulo.monitor.rest.gc;
 
 import org.apache.accumulo.core.gc.thrift.GcCycleStats;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * Metrics about a single cycle of the garbage collector
+ * Metrics about one type of garbage collection
  *
- * @since 2.0.0
+ * @since 2.1.0
```

Comment:
You reverted the other `@since` tag. Did you mean to leave this one? Not sure if you had different rationale for this one.

Reply:
Yeah, I renamed and changed the class so didn't think the old tag applied anymore. We probably should just remove some of these tags...